workspaces/announcements/.changeset/config.json

```diff
@@ -1,5 +1,5 @@
 {
-  "$schema": "https://unpkg.com/@changesets/config@3.0.0/schema.json",
+  "$schema": "https://unpkg.com/@changesets/config@3.0.5/schema.json",
```

Comment:
Not sure about this change, from my quick scan all the others are still `3.0.0` 🤔

Reply:
Reverted.